lib/beaker/dsl/helpers/web_helpers.rb

```diff
@@ -92,7 +92,7 @@ def fetch_http_file(base_url, file_name, dst_dir)
         # @!visibility private
         def fetch_http_dir(url, dst_dir)
           logger.notify "fetch_http_dir (url: #{url}, dst_dir #{dst_dir})"
-          if !/\//.match?(url[-1, 1])
+          if !url[-1, 1].include?('/')
```

Comment:
Isn't this essentially:
```suggestion
          if !url.end_with('/')
```

Reply:
Good call!